source/getting-started/sec-data-types.ptx

```diff
@@ -85,7 +85,7 @@
   </sage>
 
   <p>
-    <term>set</term>: Collection of unique items within a pair of curly braces <c>{}</c>. <c>set()</c> with lowercase <c>s</c> are built into Python. Every item in a set is unique. If you define a set with duplicate items, the duplicates will be removed.
+    <term>set</term>: Collection of unique items within a pair of curly braces <c>{}</c>. <c>set()</c> with lowercase <c>s</c> are built into Python. Every item in a set is unique.
```

Comment:
"<c>set()</c> with lowercase <c>s</c> are built into Python."

Should be "is built into".